lib/ernest.rb

```diff
@@ -62,19 +62,26 @@ def valid_key?
   get '/addresses' do
     content_type :json
 
-    page = Address.page(params[:page].to_i)
+    if params[:updated_since]
+      return 400 if !params[:updated_since].match(/[0-9]{4}\-[0-9]{2}\-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\+[0-9]{4}/)
```

Comment:
sorry, this is probably really annoying, but this won't support partial ISO8601, or Z time zones. We can probably be a bit less strict here and just do a DateTime.parse; if that's not nil, then we're OK to go.